image_test.go

```diff
@@ -367,6 +368,55 @@ func TestImageRW(t *testing.T) {
 	}
 }
 
+func TestImageCubeRoundTrip(t *testing.T) {
+	strides := []int{2, 3, 6} // non repretitive, aperiodic strides
+
+	// the data looks like [0, 1, 2, ...]
+	data := make([]int16, 2*3*6)
+	for i := 0; i < len(data); i++ {
+		data[i] = int16(i - 32768)
+	}
+	var b bytes.Buffer
+	fits, err := Create(&b)
+	if err != nil {
+		t.Error(err)
+	}
+	im := NewImage(16, strides)
+	cards := []Card{
+		{Name: "BZERO", Value: 32768},
+		{Name: "BSCALE", Value: 1.}}
+	im.Header().Append(cards...)
+	err = im.Write(data)
+	if err != nil {
+		t.Error(err)
+	}
+	err = fits.Write(im)
+	if err != nil {
+		t.Error(err)
+	}
+	im.Close()
+	fits.Close()
```

Comment:
we should probably test the returned errors. (of `im.Close()` and `fits.Close()`)

Reply:
`imageHDU.Close()` just returns nil and I didn't find any other lines that call `Close` in the test that check the error, do you think it's necessary?

Reply:
`astrogo/fitsio` was one of my first Go packages, so it's probably not up to the kind of standard I am now expecting Go code to be :)
as tests are a possible source of copy-paste-for-examples (or plain user code), I'd prefer it to be as correct as possible.

alternatively:
```
_ = im.Close()
```
to be explicit. (and alert the future reader about it).

Reply:
I'm inclined to leave it unchecked.  The Goland linter doesn't like doing so, but in my experience it's pretty universal to see Close errors unchecked -- so much so that there are [blog posts](https://www.joeshaw.org/dont-defer-close-on-writable-files/) about it :)

It would be better, IMO, to merge these changes with the error unchecked and have a separate PR that cleans up the test suite -- checking here would be inconsistent with the other test, which I think would be more confusing to someone reading for examples than the alternative.  I think the reader expects any `Close` function (in or outside of fitsio) to implement io.Closer, so they would be expecting an error... and ignoring it.

Your call.